docs/reference/errors-and-warnings/NU1504.md

```diff
@@ -0,0 +1,43 @@
+---
+title: NuGet Error NU1504
+description: NU1504 error code
+author: nkolev92
+ms.author: nikolev
+ms.date: 03/31/2022
+ms.topic: reference
+f1_keywords: 
+  - "NU1504"
+---
+
+# NuGet Error NU1504
+
+## Scenario 1
+
+<pre>Duplicate 'PackageReference' items found. Remove the duplicate items or use the Update functionality to ensure a consistent restore behavior. The duplicate 'PackageReference' items are: X 1.0.0, X 2.0.0.</pre>
+
+### Issue
+
+Sometimes when adding `PackageReference` items, you may end up adding an item for a particular package multiple times.

```

Comment:
I tried to be more concise. Here's my suggestion:

```suggestion
This error is raised when adding multiple `PackageReference` items, in the same or in multiple project files.
```

Reply:
I appreciate the suggestion, but that text would effective duplicate the scenario message. 

The description is meant to explain how you may have run into the scenario.